src/main/java/seedu/address/model/statistics/StatisticData.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.model.statistics;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import javafx.scene.chart.PieChart.Data;
+
+/**
+ * Represents a data point in the Statistical Data for a PieChart.
+ * Has a String {@code name} and double {@code value} as it is a piechart datapoint.
+ * Guarrantees: Immutable, data points are not null.
+ */
+public class StatisticData {
+
+    //PieChart data point.
+    private final Data chartData;
+
+    /**
+     * Constructs a {@code StatisticData} with the given {@code name} and {@code value}.
+     * {@code name} and {@code value} must not be null.
```

Comment:
```suggestion
```
Can be removed

Reply:
oh but this one is like consistently in the other similar model classes as well